src/tooling/piral-cli/src/common/config.ts

```diff
@@ -53,4 +74,9 @@ export const config: PiralCliConfig = rc('piral', {
   piletApi: '/$pilet-api',
   validators: {},
   schemaVersion: 'v2',
+  openBrowser: false,
+  port: 1234,
+  language: SourceLanguage.ts,
+  host: 'localhost',
+  registry: 'https://registry.npmjs.org/',
```

Comment:
We have `defaultRegistry` already. I'd say let's use this one instead of hardcoding it again.

Reply:
The `registry` was moved from [`piral-cli/src/common/constants.ts`](https://github.com/smapiot/piral/blob/f1f2eca51d8342cd2d16718eb4759c83aba1979b/src/tooling/piral-cli/src/common/constants.ts)(The original `defaultRegistry` has been deleted), do you mean to move `registry` back?